### ISSUE TITLE
Package custom pieces for easier install

### ISSUE THREAD

**Issue by dubrod** [modxcms/a11y#29]
https://github.com/modxcms/a11y/blob/master/Installation.md

the install process needs compiling

**Comment by rtripault** [comment]
What about releasing "vertical navigation" as a separate/standalone component and make it a dependency for a11y ? That would be a nice way to start playing with newly introduced dependencies...

**Comment by dubrod** [comment]
this needs to be a stand alone installer/theme package that might ship with a core release or might be an additional add-on via install/cloud setup.

**Comment by dubrod** [comment]
I removed the plugin, namespace, system setting and rolled in the CSS and layout.js is functioning already without the vertical-navigation.js. I think it might be missing a little bit of functionality like keyboard nav, so you can add that back in or leave it alone b/c we need to combine the aria-tree problem as well. 
Either way Installation is a little easier.

**Comment by rtripault** [comment]
First step done : http://modx.com/extras/package/verticalnavigation / https://github.com/meltingmedia/Vertical-Navigation

Now to package the theme!

**Comment by dubrod** [comment]
@TheBoxer lets get one of your fancy transport packages started before it gets too late and so we can spin up test sites easier. thanks!

**Comment by dubrod** [comment]
this is no longer urgent we are down to 4 easy steps but the final install is still up in the air

**Comment by dubrod** [comment]
i think we should actually move this to the very end. its still very unclear how we are going to solve all these issues so keeping up with an install package would be a lot of redundant tweaking.
